test/extra/lxml_test.py

```diff
@@ -13,11 +13,7 @@ def install_and_test_lxml():
     os.makedirs(SRC_DIR)
 
     url = "https://github.com/cython/cython"
-    subprocess.check_call(["git", "clone", "--depth", "1", "--branch", "0.24", url], cwd=SRC_DIR)
-
-    PATCH_FILE = os.path.abspath(os.path.join(os.path.dirname(__file__), "../integration/Cython-0.24.patch"))
-    subprocess.check_call(["patch", "-p1", "--input=" + PATCH_FILE], cwd=CYTHON_DIR)
-    print ">>> Applied Cython patch"
+    subprocess.check_call(["git", "clone", "--depth", "1", "--branch", "0.25a0", url], cwd=SRC_DIR)
```

Comment:
FYI it looks like they uploaded this to pypi as well so we should be able to do `pip install cython==0.25a0`!